developer-docs/updating_rke2_charts.md

```diff
@@ -0,0 +1,89 @@
+# Updating RKE2 charts
+
+In each release, it is very common to update the charts that rke2 is using. This document describes the process that must be followed to succeed in updating and integrating it into rke2 and Rancher.
```

Comment:
Could you please limit the line length to some reasonable amount - 80 or 100? It seems like it's done in all the other .md files and IMO it's way better for text editors (easier to find some particular word in a particular line, easier to jump over with vim shortcuts, looks nicer).

Reply:
Here is the example of other md file:

https://raw.githubusercontent.com/rancher/rke2/master/BUILDING.md

The limit seems to be 120. So maybe let's stick to 120?

I'm fine with links exceedeing the limit. Or in general, I don't mind few exceptions of a bit longer lines, as long as they're an exception rather than a rule. :) But I'm against putting whole paragraphs in single lines.